rpm/syslinux.spec

```diff
@@ -122,7 +124,9 @@ mkdir -p %{buildroot}/etc
 # If we have a /boot/extlinux.conf file, assume extlinux is our bootloader
 # and update it.
 if [ -f /boot/extlinux/extlinux.conf ]; then \
-    extlinux --update /boot/extlinux ; \
+    MOUNTPOINT=$(stat -c "%m" /boot/extlinux)
+    DEVICE=$(findmnt -no SOURCE $MOUNTPOINT)
```

Comment:
I think this would be easier to read if you use the long options instead of the short ones.

Reply:
changed to long options